src/grafana_dashboards/metrics.json

```diff
@@ -289,6 +247,18 @@
           "legendFormat": "Idle Runners",
           "queryType": "range",
           "refId": "D"
+        },
+        {
+          "datasource": {
+            "type": "loki",
+            "uid": "${lokids}"
+          },
+          "editorMode": "code",
+          "expr": "sum by(filename)(last_over_time({filename=\"/var/log/github-runner-metrics.log\", juju_application=~\"$juju_application\", juju_model=~\"$juju_model\", juju_model_uuid=~\"$juju_model_uuid\", juju_unit=~\"$juju_unit\"} | json event=\"event\",active_runners=\"active_runners\" | event=\"reconciliation\" | unwrap active_runners[10m]))",
+          "hide": false,
+          "legendFormat": "Active Runners",
+          "queryType": "range",
+          "refId": "A"
         }
       ],
       "title": "Gauges",
```

Comment:
I think this title is a bit awkward. I'm not sure what a better title would be (possibly "Active and Idle Runners"), since we already have a "Runners Total" graph as well. If we do call this "Active and Idle Runners" is it clear how that's different from "Runners Total"?

Reply:
We can change it to 'Active and Idle Runners' and perhaps reword 'Runners Total' to 'Lifecycle Status'?